tests/jeos/firstrun.pm

```diff
@@ -405,6 +405,16 @@ sub run {
     verify_norepos unless is_opensuse;
     verify_bsc if is_jeos;
     verify_partition_label;
+
+    if (has_selinux_by_default) {
+        validate_script_output("sestatus", sub { m/SELinux status: .*enabled/ });
+    } else {
+        if (script_run("which sestatus")) {
```

Comment:
You could also do `grep -qw selinux /sys/kernel/security/lsm`, but both approaches are valid IMO

Reply:
Indeed! I forgot about that one, it's way cleaner.